src/test/com/artos/testcase/TestCase_2.java

```diff
@@ -0,0 +1,19 @@
+package test.com.artos.testcase;
+
+import com.artos.annotation.TestCase;
+import com.artos.annotation.TestPlan;
+import com.artos.annotation.Unit;
+import com.artos.framework.infra.TestContext;
+import com.artos.interfaces.TestExecutable;
+
+@TestPlan(preparedBy = "Alan", preparationDate = "2019/11/6", bdd = "GIVEN..WHEN..AND..THEN..")
+@TestCase
+public class TestCase_2 implements TestExecutable {
+
+    @Unit
+    public void testUnit_2_1(TestContext context) {
+        // --------------------------------------------------------------------------------------------
+        System.out.println("testUnit_2_1");
+        // --------------------------------------------------------------------------------------------
+    }
+}
```

Comment:
What does this test achieves?

Reply:
Because if there is only one test case, then the GUI selector will execute directly instead of let the user select, so I created the 2nd dummy test case. 
Actually the test case list only need more then one object, and they don't need to be different classes. So technically we can define only one dummy class. I create this 2nd one just for a different display name on the GUI selector list.